remotespark/wrapperkernel/codetransformers.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+from remotespark.utils.constants import Constants
+
+
+class UserCodeTransformerBase(object):
+    def __init__(self, subcommand):
+        self.name = subcommand
+
+    def get_code_to_execute(self, session_started, connection_string, force, output_var, command):
+        raise NotImplementedError()
+
+
+class NotSupportedTransformer(UserCodeTransformerBase):
+    def get_code_to_execute(self, session_started, connection_string, force, output_var, command):
+        error_to_show = "Magic '{}' not supported.".format(self.name)
+        code_to_run = ""
+        begin_action = Constants.do_nothing_action
+        end_action = Constants.do_nothing_action
+        deletes_session = False
+
+        return code_to_run, error_to_show, begin_action, end_action, deletes_session
```

Comment:
I'm a little bit ambivalent about returning 5 results from this function (I'm definitely not going to remember the order, what if they get mixed up?)

Maybe returning an instance of a class or a namedtuple would be better here.